db/migrations/0001_initial.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 4.0.2 on 2024-12-11 21:12
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Guild',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255, unique=True)),
+                ('description', models.TextField(blank=True)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Race',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255, unique=True)),
+                ('description', models.TextField(blank=True)),
+                ('race', models.CharField(choices=[('elf', 'Elf'), ('dwarf', 'Dwarf'), ('human', 'Human'), ('ork', 'Ork')], max_length=20)),
```

Comment:
The `race` field with `STATUS_CHOICES` in the `Race` model is not required by the task and should be removed to align with the requirements.